x/go/testutil/cesium_test_utils.go

```diff
@@ -0,0 +1,13 @@
+package testutil
+
+import (
+	"github.com/synnaxlabs/cesium"
+)
+
+var k cesium.ChannelKey = 1
+
+func GenerateCesiumChannelKey() (key cesium.ChannelKey) {
+	key = k
+	k++
+	return
```

Comment:
code cov

Reply:
This function will eventually fuck us if we try to run these tests concurrently. Just something to be aware of

Reply:
Yeah made this atomic